### PR TITLE
[Frontend][Paddle] [PaddlePaddle Hackathon 4]add attribute support for gaussian_random/softplus/Conv3d/Conv2d

### PR DESCRIPTION
**Pull request by TheFormerWalker**
add dtype attribute for gaussian_random op.
add threshold attribute for softplus op.
add data_format attribute for Conv3d/Conv2d op.

### CODE REVIEW

File: python/tvm/relay/frontend/paddlepaddle.py [hunk 1]
```diff
@@ -2173,7 +2178,14 @@ def convert_softplus(g, op, block):
...
     beta = _expr.const(beta, dtype=dtype)
-    out = _op.log(_op.exp(x * beta) + _expr.const(1.0, dtype=dtype)) / beta
+    threshold = op.attr("threshold")
+    if threshold is None:
+        threshold=_expr.const(20.0, dtype=dtype)
+    threshold = _expr.const(threshold, dtype=dtype)
+    if x*beta <= threshold:
+       out = _op.log(_op.exp(x * beta) + _expr.const(1.0, dtype=dtype)) / beta
+    else:
+       out=x
```
Comment (jiangjiajun) [inline python/tvm/relay/frontend/paddlepaddle.py:2188]: Please add test for this attribute

File: tests/python/frontend/paddlepaddle/test_forward.py [hunk 2]
```diff
@@ -595,6 +633,7 @@ def forward(self, inputs):
             input_data=input_data,
         )
         verify_model(Conv3D(stride=2, padding="SAME", dilation=2, groups=3), input_data=input_data)
+        verify_model(Conv3D2(stride=2, padding="SAME", dilation=2, groups=3, data_layout='NHWC'), input_data=input_data)
```
Comment (jiangjiajun) [inline tests/python/frontend/paddlepaddle/test_forward.py:636]: It should be 'NCDHW'

**Review by jiangjiajun — CHANGES_REQUESTED** [review]

File: python/tvm/relay/frontend/paddlepaddle.py (hunk 1 above)
Comment (TheFormerWalker) [reply python/tvm/relay/frontend/paddlepaddle.py:2188]: Done

File: tests/python/frontend/paddlepaddle/test_forward.py (hunk 2 above)
Comment (TheFormerWalker) [reply tests/python/frontend/paddlepaddle/test_forward.py]: Done

**Review by jiangjiajun — APPROVED** [review]
LGTM

**Review by jiangjiajun — APPROVED** [review]
LGTM

**Comment by jiangjiajun** [comment]
Hi， @junrushao, Could you help to merge this PR?

**Comment by junrushao** [comment]
Hi @jiangjiajun, I am happy to but seems that the linter fails at the moment. Please ping me after the PR passes CI :-)

**Comment by jiangjiajun** [comment]
@TheFormerWalker

**Comment by TheFormerWalker** [comment]
done

**Comment by TheFormerWalker** [comment]
@jiangjiajun Done.

**Review by jiangjiajun — APPROVED** [review]
LGTM

**Review by jiangjiajun — APPROVED** [review]
There are still 4 checks not passed, please fix them, if not related with your pull request, try to trigger the checks again.

**Comment by TheFormerWalker** [comment]
> There are still 4 checks not passed, please fix them, if not related with your pull request, try to trigger the checks again.

They are not related with my pull request, I don‘t konw how to fix them.

**Comment by TheFormerWalker** [comment]
> There are still 4 checks not passed, please fix them, if not related with your pull request, try to trigger the checks again.
@jiangjiajun Done. I‘m sorry that I don't have enough time to fix the gpu CI error of  test_forward_gaussian_random, so I delete the part of test_forward_gaussian_random.

**Review by jiangjiajun — APPROVED** [review]
Thanks for your contribution. LGTM

**Review by jiangjiajun — APPROVED** [review]
Thanks for your contribution. LGTM @junrushao

**Pull request merged**